docs/base.html

```diff
@@ -26,18 +26,123 @@ <h1 class="title">Module <code>keboola.component.base</code></h1>
 <summary>
 <span>Expand source code</span>
 </summary>
-<pre><code class="python">import logging
-import os
+<pre><code class="python">import contextlib
 import json
-from . import dao
-from . import table_schema as ts
+import logging
+import os
+import sys
 from abc import ABC, abstractmethod
+from functools import wraps
 from pathlib import Path
 from typing import Optional, Dict
+
+from . import dao
+from . import table_schema as ts
 from .interface import CommonInterface
 
 KEY_DEBUG = &#39;debug&#39;
 
+# Mapping of sync actions &#34;action name&#34;:&#34;method_name&#34;
+_SYNC_ACTION_MAPPING = {&#34;run&#34;: &#34;run&#34;}
+
+
+def sync_action(action_name: str):
+    &#34;&#34;&#34;
```

Comment:
is this a valid replacement for the quotation marks?

Reply:
It's a html code. This is generated by the documentation module. Preview here https://htmlpreview.github.io/?https://raw.githubusercontent.com/keboola/python-component/KCOFAC-1703-Sync-Actions-as-decorator/docs/base.html